### PR TITLE
Add Check

### PR DESCRIPTION
**Pull request by satmandu**
- newer pulseaudio asks for this.

Works properly:
- [x] x86_64
- [x] i686
- [x] armv7l

### CODE REVIEW

File: packages/check.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  })
+
+
+  def self.build
+    system "autoreconf -fvi"
+    Dir.mkdir 'build'
+    Dir.chdir 'build' do
+      system "env CC=gcc LD=ld \
+      cmake -G Ninja \
+      -DCMAKE_INSTALL_PREFIX=/usr/local \
```
Comment (uberhacker) [inline packages/check.rb:30]: Abstract with `#{CREW_PREFIX}` or better yet, `#{CREW_CMAKE_OPTIONS}`?

Comment (satmandu) [reply packages/check.rb:30]: I tried that and I was getting a bunch of errors, which is why I just went back to the minimal options which worked.

Comment (uberhacker) [reply packages/check.rb:30]: That is fine.  You don't have to recreate the binaries but please change `/usr/local` to `#{CREW_PREFIX}`.

Comment (satmandu) [reply packages/check.rb:30]: Whoops! Good catch.

**Pull request merged**